modules/nf-core/modules/svdb/merge/main.nf

```diff
@@ -37,6 +36,7 @@ process SVDB_MERGE {
         $prio \\
         --vcf $input \\
         > ${prefix}_sv_merge.vcf
+    bgzip ${prefix}_sv_merge.vcf
 
```

Comment:
Ah just saw that you're not adding the samtools version to the `versions.yml`

Reply:
Can you fix that @WackerO ?